### PR TITLE
Add pool question

### PR DESCRIPTION
**Pull request by darkterreur**
This Pull Request contains: 
- the PoolQuestion CRUD without linking to the questions
- the Question CRUD with linking to the PoolQuestion and without linking Response
- the Reponse CRUD with linking to the Question and without linking video

### CODE REVIEW

File: src/AppBundle/Controller/BackendController.php [hunk 1]
```diff
@@ -3,210 +3,10 @@
...
-use Symfony\Component\HttpFoundation\Request;
-use AppBundle\Entity\PoolVideo;
-use AppBundle\Form\PoolVideoType;
-use AppBundle\Entity\Video;
-use AppBundle\Form\VideoType;
-use AppBundle\Form\ModifVideoType;
+
 
 class BackendController extends Controller
 {
```
Comment (geoffrey-chameroy) [inline src/AppBundle/Controller/BackendController.php]: Delete this file if there is no function inside

File: src/AppBundle/Controller/BackendPoolVideo.php [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+                return $this->redirectToRoute('pool_video_manage');
+            } else {
+                $request->getSession()
+                    ->getFlashBag()
+                    ->add('danger', 'Formulaire non valide');
+            }
+        }
+
+        return $this->render('Backend/modifPoolVideo.html.twig', array('form' => $form->createView()));
```
Comment (geoffrey-chameroy) [inline src/AppBundle/Controller/BackendPoolVideo.php]: Go to line.

File: src/AppBundle/Controller/BackendQuestionController.php [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+                return $this->redirectToRoute('question_manage');
+            } else {
+                $request->getSession()
+                    ->getFlashBag()
+                    ->add('danger', 'Formulaire non valide');
+            }
+        }
+
+        return $this->render('Backend/addQuestion.html.twig', array('form' => $form->createView()));
```
Comment (geoffrey-chameroy) [inline src/AppBundle/Controller/BackendQuestionController.php]: Go to line.

File: src/AppBundle/Controller/BackendQuestionController.php [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+                    ->add('success', 'Question bien modifiée');
+                return $this->redirectToRoute('question_manage');
+            } else {
+                $request->getSession()
+                    ->getFlashBag()
+                    ->add('danger', 'Formulaire non valide');
+            }
+        }
+
+        return $this->render('Backend/modifQuestion.html.twig', array('form' => $form->createView()));
```
Comment (geoffrey-chameroy) [inline src/AppBundle/Controller/BackendQuestionController.php]: Go to line.

File: src/AppBundle/Controller/BackendReponseController.php [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+                return $this->redirectToRoute('reponse_manage');
+            } else {
+                $request->getSession()
+                    ->getFlashBag()
+                    ->add('danger', 'Formulaire non valide');
+            }
+        }
+
+        return $this->render('Backend/addReponse.html.twig', array('form' => $form->createView()));
```
Comment (geoffrey-chameroy) [inline src/AppBundle/Controller/BackendReponseController.php]: Go to line.

File: src/AppBundle/Controller/BackendReponseController.php [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+                    ->add('success', 'Reponse bien modifiée');
+                return $this->redirectToRoute('reponse_manage');
+            } else {
+                $request->getSession()
+                    ->getFlashBag()
+                    ->add('danger', 'Formulaire non valide');
+            }
+        }
+
+        return $this->render('Backend/modifReponse.html.twig', array('form' => $form->createView()));
```
Comment (geoffrey-chameroy) [inline src/AppBundle/Controller/BackendReponseController.php]: Go to line.

File: src/AppBundle/Controller/BackendVideoController.php [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+                return $this->redirectToRoute('video_manage');
+            } else {
+                $request->getSession()
+                    ->getFlashBag()
+                    ->add('danger', 'Formulaire non valide');
+            }
+        }
+
+        return $this->render('Backend/addVideo.html.twig', array('form' => $form->createView()));
```
Comment (geoffrey-chameroy) [inline src/AppBundle/Controller/BackendVideoController.php]: Go to line.

File: src/AppBundle/Controller/BackendVideoController.php [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+                    ->add('success', 'Video bien modifiée');
+                return $this->redirectToRoute('video_manage');
+            } else {
+                $request->getSession()
+                    ->getFlashBag()
+                    ->add('danger', 'Formulaire non valide');
+            }
+        }
+
+        return $this->render('Backend/modifVideo.html.twig', array('form' => $form->createView()));
```
Comment (geoffrey-chameroy) [inline src/AppBundle/Controller/BackendVideoController.php]: Go to line.

File: src/AppBundle/Entity/Reponse.php [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+     * @ORM\Column(name="libelle", type="string", length=255)
+     */
+    private $libelle;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="fair", type="boolean")
+     */
+    private $fair;
```
Comment (geoffrey-chameroy) [inline src/AppBundle/Entity/Reponse.php]: use is_fair for a boolean

File: app/Resources/views/Backend/listeVideo.html.twig [hunk 10]
```diff
@@ -12,21 +12,20 @@
                     <td>Titre</td>
                     <td>Extension</td>
                     <td>Description</td>
-                    <td>
+                     <td width="100px" align="center">
```
Comment (geoffrey-chameroy) [inline app/Resources/views/Backend/listeVideo.html.twig]: You should use the bootstrap class "text-center"

File: src/AppBundle/Controller/BackendPoolQuestionController.php [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+
+class BackendPoolQuestionController extends Controller
+{
+    public function managePoolQuestionAction()
+    {
+        $listPoolQuestion = $this->getDoctrine()->getManager()
+            ->getRepository('AppBundle:PoolQuestion')
+            ->findAll();
+
+        return $this->render('Backend/listePoolQuestion.html.twig', array(
```
Comment (geoffrey-chameroy) [inline src/AppBundle/Controller/BackendPoolQuestionController.php]: Filename in mi-french

File: src/AppBundle/Controller/BackendPoolVideoController.php [hunk 12]
```diff
@@ -0,0 +1,114 @@
...
+
+class BackendPoolVideoController extends Controller
+{
+    public function managePoolVideoAction()
+    {
+        $listPoolVideo = $this->getDoctrine()->getManager()
+            ->getRepository('AppBundle:PoolVideo')
+            ->findAll();
+
+        return $this->render('Backend/listePoolVideo.html.twig', array(
```
Comment (geoffrey-chameroy) [inline src/AppBundle/Controller/BackendPoolVideoController.php]: Filename in mi-french

File: src/AppBundle/Controller/BackendQuestionController.php [hunk 13]
```diff
@@ -0,0 +1,111 @@
...
+
+class BackendQuestionController extends Controller
+{
+    public function manageQuestionAction()
+    {
+        $listQuestion = $this->getDoctrine()->getManager()
+            ->getRepository('AppBundle:Question')
+            ->findAll();
+
+        return $this->render('Backend/listeQuestion.html.twig', array(
```
Comment (geoffrey-chameroy) [inline src/AppBundle/Controller/BackendQuestionController.php]: Filename in mi-french

File: src/AppBundle/Controller/BackendResponseController.php [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+
+class BackendResponseController extends Controller
+{
+    public function manageResponseAction()
+    {
+        $listResponse = $this->getDoctrine()->getManager()
+            ->getRepository('AppBundle:Response')
+            ->findAll();
+
+        return $this->render('Backend/listeResponse.html.twig', array(
```
Comment (geoffrey-chameroy) [inline src/AppBundle/Controller/BackendResponseController.php]: Filename in mi-french

File: src/AppBundle/Controller/BackendVideoController.php [hunk 15]
```diff
@@ -0,0 +1,109 @@
...
+
+class BackendVideoController extends Controller
+{
+    public function manageVideoAction()
+    {
+        $listVideo = $this->getDoctrine()->getManager()
+            ->getRepository('AppBundle:Video')
+            ->findAll();
+
+        return $this->render('Backend/listeVideo.html.twig', array(
```
Comment (geoffrey-chameroy) [inline src/AppBundle/Controller/BackendVideoController.php]: Filename in mi-french

File: src/AppBundle/Entity/Question.php [hunk 16]
```diff
@@ -0,0 +1,314 @@
...
+
+
+    /**
+     * Set poolQuestion
+     *
+     * @param \stdClass $poolQuestion
+     *
+     * @return Video
+     */
+    public function setPoolQuestion($poolQuestion)
```
Comment (geoffrey-chameroy) [inline src/AppBundle/Entity/Question.php]: Must be check with an instance of PoolQuestion

Comment (alcalyn) [inline src/AppBundle/Entity/Question.php]: http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration

**Pull request merged**